common/src/main/java/com/wynntils/services/mapdata/attributes/impl/OffsetMapVisiblity.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright © Wynntils 2024.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.services.mapdata.attributes.impl;
+
+import com.wynntils.services.mapdata.attributes.type.MapVisibility;
+
+public class OffsetMapVisiblity implements MapVisibility {
```

Comment:
Given our discussion in discord, I think this class should basically be replaced by something that takes in a min, max and fade values, and interpolates based on them. I'm not sure if we should remove the interface, I think it could be good to keep it since that leaves open for hard-coded (non-json-based) internal mapdata providers behaving differently. But we should in practice only use the min-max-fade based one.